lumicks/pylake/fitting/fit.py

```diff
@@ -325,8 +329,11 @@ def profile_likelihood(
         if self.params[parameter_name].fixed:
             raise RuntimeError(f"Parameter {parameter_name} is fixed in the fitting object.")
 
-        assert max_step > min_step
-        assert max_chi2_step > min_chi2_step
+        if max_step <= min_step:
+            raise ValueError("max_step must be larger than min_step")
```

Comment:
I think it's always a good practice to show the user the values that are passed to the function, to better understand why it's broken 

```suggestion
            raise ValueError(f"max_step must be larger than min_step, got max_step={max_step} and min_step={min_step}")
```

Reply:
Absolutely. Good catch! 👍